package/network/services/dnsmasq/Makefile

```diff
@@ -10,7 +10,7 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=dnsmasq
 PKG_UPSTREAM_VERSION:=2.89
 PKG_VERSION:=$(subst test,~~test,$(subst rc,~rc,$(PKG_UPSTREAM_VERSION)))
-PKG_RELEASE:=6
+PKG_RELEASE:=7
```

Comment:
You can skip bumping the release as this fix a compile warning. (no idea why it does say it does block imagebuilder)

Reply:
> You can skip bumping the release as this fix a compile warning. (no idea why it does say it does block imagebuilder)

The issue reported related to Image Builder builds when enabling the dnsmasq service. Does this issue show up during compile time??

Reply:
I don’t use ImageBuilder so I can’t say.

Reply:
To my knowledge, it shouldn't make a difference at runtime - unless you're using `$IPKG_INSTROOT` on the router.

Reply:
Done